### PR TITLE
feat: Adding PolyUtil extensions to List<LatLng>

### PR DESCRIPTION
**Pull request by arriolac**
`Polygon.kt` already contain these extensions but these should be added to `List<LatLng>` extensions as well.

### CODE REVIEW

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt [hunk 1]
```diff
@@ -15,12 +15,65 @@
...
 @file:Suppress("NOTHING_TO_INLINE")
+
 package com.google.maps.android.ktx
 
 import com.google.android.gms.maps.model.LatLng
 import com.google.maps.android.PolyUtil
 import com.google.maps.android.SphericalUtil
 
+/**
+ * Computes where the given [latLng] is contained on or near this Polyline within a specified
```
Comment (barbeau) [inline maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: `where the given [latLng] is contained on or near` -> `whether the given [latLng] lies on or near`

Comment (barbeau) [reply maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: should `this` be `[this]`? I'm not sure how `this` works with Kotlin docs.

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt [hunk 2]
```diff
@@ -15,12 +15,65 @@
...
+ * @param geodesic if this polyline is geodesic or not
+ * @param tolerance the tolerance in meters
+ * @return true if [latLng] is on this path, otherwise, false
+ *
+ * @see PolyUtil.isLocationOnPath
+ */
+inline fun List<LatLng>.isLocationOnPath(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
```
Comment (barbeau) [inline maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: `0.1` should be `DEFAULT_TOLERANCE` from the Java lib

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt [hunk 3]
```diff
@@ -15,12 +15,65 @@
...
+ * @param geodesic if this polygon is geodesic or not
+ * @param tolerance the tolerance in meters
+ * @return true if [latLng] lies on or is near the edge of this Polygon, otherwise, false
+ *
+ * @see PolyUtil.isLocationOnEdge
+ */
+inline fun List<LatLng>.isOnEdge(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
```
Comment (barbeau) [inline maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: `0.1` should be `DEFAULT_TOLERANCE` from the Java lib

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt [hunk 4]
```diff
@@ -15,12 +15,65 @@
...
+ * @see PolyUtil.isLocationOnPath
+ */
+inline fun List<LatLng>.isLocationOnPath(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
+): Boolean = PolyUtil.isLocationOnPath(latLng, this, geodesic, tolerance)
+
+/**
+ * Checks whether or not [latLng] lies on or is near the edge of this polygon within a tolerance
```
Comment (barbeau) [inline maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: `tolerance` is repeated here - maybe `within the [tolerance] (in meters)`?

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt [hunk 5]
```diff
@@ -15,12 +15,65 @@
...
+ * @see PolyUtil.isLocationOnEdge
+ */
+inline fun List<LatLng>.isOnEdge(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
+): Boolean = PolyUtil.isLocationOnEdge(latLng, this, geodesic, tolerance)
+
+/**
+ * Computes whether the given point lies inside the specified polygon.
```
Comment (barbeau) [inline maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: `point` -> `latLng`
`the specified` -> `this`

**Comment by arriolac** [comment]
> Also, is Codecov not working? 

@barbeau yep, it's not working for some reason. This issue is open #8

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt (hunk 3 above)
Comment (arriolac) [reply maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: That's what I meant to do now that AMU 1.0.1 is live on maven. I'll need to bump the version used by the KTX lib and update this.

Comment (arriolac) [reply maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: Actually I'll address this as a separate PR (with version bumping AMU to 1.0.1). Will update the PR with doc edits though.

File: maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt (hunk 1 above)
Comment (arriolac) [reply maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt]: Just tested—`[this]` doesn't work in Kotlin docs

**Pull request merged**